gh_actions_exporter/metrics.py

```diff
@@ -229,11 +229,19 @@ def flavor_type(self, webhook: WebHook) -> str or None:
                 return label
         return None
 
+    def get_job_cost(self, workflow_job: WorkflowJob, flavor_label: str) -> float:
+        cost_per_min: float = self.settings.job_costs.get(
+            flavor_label, self.settings.default_cost
+        )
+        duration: int = int(workflow_job.completed_at.timestamp()) - int(
+            workflow_job.started_at.timestamp()
+        )
+
+        return duration / 60 * cost_per_min
+
```

Comment:
Why are you not using `Cost.get_job_cost`?

Reply:
Oups, I forgot ! I change this